spec/integrations/sidekiq_spec.rb

```diff
@@ -29,6 +30,8 @@ def perform(value)
         expect(event["metaData"]["sidekiq"]["msg"]["args"]).to eq([-0])
         expect(event["metaData"]["sidekiq"]["msg"]["queue"]).to eq("default")
         expect(event["severity"]).to eq("error")
+        expect(event["app"]["type"]).to eq("sidekiq")
+        expect(event["device"]["runtimeVersions"]["sidekiq"]).to match(/\A\d+\.\d+\.\d+/)
```

Comment:
It turns out I was wrong at https://github.com/bugsnag/bugsnag-ruby/pull/560#discussion_r314879567.

The version is stubbed via `stub_const` on line 18, so this should probably be something like `.to eq('2.0.0')` instead.